README.md

```diff
@@ -1 +1,41 @@
-# visual-studio-code-extension-project
\ No newline at end of file
+
+# TYPESCRIPT AST PARSER VISUAL STUDIO CODE EXTENSION
+
+This Visual Studio extension does static analysis on TypeScript code to help people during development.
+
+## Installation
+
+*Option 1:*
+
+* Open Visual Studio
+
+* Click Extensions button on toolbar on left of screen or press Ctrl+Shift+X
+
+* In search box type 'medFive'
+
+* Typescript Static Anaylsis should be returned as result
+
+* Presss 'Install'
+
+*Option 2:*
+
+* Open web browser
+
+* Navigate to https://marketplace.visualstudio.com/items?itemName=medFive.typescript-static-analysis
+
+* Press 'Install'
+
+## How to Use
+
+* Hover over code and result will display with attributes
+
+## Test Case
+
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
+
+## License
+[MIT](https://choosealicense.com/licenses/mit/)
```

Comment:
We are not using MIT as our license
Unlicensed is our license, you can add this link: https://unlicense.org